tensorpack/callbacks/saver.py

```diff
@@ -78,6 +78,7 @@ def _trigger_epoch(self):
                 self.path,
                 global_step=get_global_step(),
                 write_meta_graph=False)
+            logger.info("Model saved to %s" % (self.path))
```

Comment:
Maybe you'd like self.checkpoint_dir better?
self.path would look like "train_log/dirname/model". It's just a prefix of where the actual model is saved.

Or you can use `tf.train.get_checkpoint_state(self.checkpoint_dir).model_checkpoint_path` to get the path to the actual model.